package.json

```diff
@@ -26,6 +26,8 @@
   },
   "homepage": "https://github.com/symfony/webpack-encore",
   "dependencies": {
+    "aurelia-loader-webpack": "^2.1.0",
```

Comment:
Is this needed? Is this a loader or a plugin?

Also, these should be in `devDependencies`. The `lib/features.js` should cause the user to see a nice error that they need to install these (instead of us shipping Encore with them included)